src/formatDate.js

```diff
@@ -8,7 +8,27 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
```

Comment:
The assumption that the last element of `fromFormat` and `toFormat` is a separator might not always be correct. Consider explicitly passing the separator as a separate parameter or ensuring the format arrays are structured to always have the separator as the last element.